test/unit/helpers/views/TransportMessages.scala

```diff
@@ -19,6 +19,7 @@ package helpers.views
 trait TransportMessages {
```

Comment:
Just FYI in general we're trying to get rid of traits with messages. It's easier to inline it and use in specific places or create those variables in specific test where we use it

Specially if you're using this message inlined in the test :)